demos/camera_demo/src/keyboardorientationinput.cpp

```diff
@@ -128,11 +128,11 @@ void KeyboardOrientationInput::pollThread()
         switch (c)
         {
             case UP:
-                panTiltUpdate.tilt = -mAngIncrement;
+                panTiltUpdate.tilt = mAngIncrement;
```

Comment:
I can't tell does this change make the keyboard have a non-inverted y-axis or an inverted y-axis? In other words does the UP arrow move the camera towards the ceiling (non-inverted) or the floor (inverted)? Changing the sign will make the sliding window filter have an inverted y-axis. If the Trillium controls are not an inverted y-axis then change https://github.com/GaloisInc/pirate/blob/master/demos/camera_demo/src/xwinframeprocessor.cpp#L87 to `-position.tilt` to make the control consistent.